Assets/Scripts/FighterJet/FighterJetService.cs

```diff
@@ -2,16 +2,28 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class FighterJetService : MonoBehaviour
+namespace SkyForce.FighterJet
 {
-    private FighterJetController fighterJet;
-    void Start()
+    [RequireComponent(typeof(PlayerInputHandler))]
+    public class FighterJetService : GenericMonoSingleton<FighterJetService>
     {
-        fighterJet = new FighterJetController();
-    }
+        [SerializeField]
+        private PlayerJetScriptableObject fighterJetData;
+        private FighterJetController fighterJet;
```

Comment:
naming is off at some places its PlayerJet*** and others its FigherJet**** 
use a consistent naming pattern - Player*** is recommended so tomorrow you game changes to not be a figher jet but something else the names are still relevant in code